fapi-client/src/main/scala/com/gu/facia/api/utils/ExternalLinks.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.facia.api.utils
+
+import java.net.URI
+
+import scala.util.Try
+
+object ExternalLinks {
+  val corsOrigins = List(
```

Comment:
Why are they called this when it has nothing to do with CORS? It just serves to confuse the purpose here, which is to work out if a link is an internal Guardian one. Even inlining this list in the `GuardianDomains` property would be clearer here, I think.